perf/istio-install/setup_istio_operator.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+WD=$(dirname $0)
+WD=$(cd $WD; pwd)
+DIRNAME="${WD}/tmp"
+mkdir -p ${DIRNAME}
+export GO111MODULE=on
+
+OPERATOR_DIR="${DIRNAME}/operator"
+if [[ ! -e "$OPERATOR_DIR" ]]; then
+  git clone https://github.com/istio/operator.git "$OPERATOR_DIR"
```

Comment:
Best to do git pull on the operator dir, or specifically pin it to a sha.
Leaving it dangling is problematic.

Reply:
ok, updated the logic to pin to a sha